pkg/executables/govc.go

```diff
@@ -601,7 +596,7 @@ func (g *Govc) ValidateVCenterSetup(ctx context.Context, datacenterConfig *v1alp
 	return nil
 }
 
-func (g *Govc) ValidateVCenterSetupMachineConfig(ctx context.Context, datacenterConfig *v1alpha1.VSphereDatacenterConfig, machineConfig *v1alpha1.VSphereMachineConfig, selfSigned *bool) error {
+func (g *Govc) ValidateVCenterSetupMachineConfig(ctx context.Context, datacenterConfig *v1alpha1.VSphereDatacenterConfig, machineConfig *v1alpha1.VSphereMachineConfig, _ *bool) error {
```

Comment:
Yeah we should, I just left the `_` to indicate that is not needed
I didn't do it as part of this PR because it involves changing the mocks and the tests
It's not a lot of work but it was gonna make this PR even bigger

If you are ok with this, I prefer to make this change in separate PR